libpromises/evalfunction.c

```diff
@@ -1563,8 +1563,45 @@ static FnCallResult FnCallVariablesMatching(EvalContext *ctx, ARG_UNUSED const P
 }
 
 /*********************************************************************/
+static Bundle *GetBundleFromPolicy(const Policy *policy, const char *namespace, const char *bundlename)
+{
+    const size_t bundles = SeqLength(policy->bundles);
 
-static FnCallResult FnCallGetMetaTags(EvalContext *ctx, ARG_UNUSED const Policy *policy, const FnCall *fp, const Rlist *finalargs)
+    for (size_t i = 0; i < bundles; i++)
+    {
+        Bundle *bp = SeqAt(policy->bundles, i);
+        if ((strcmp(bp->name, bundlename) || strcmp(bp->ns, namespace)) == 0)
```

Comment:
```suggestion
        if (StringEqual(bp->name, bundlename) && StringEqual(bp->ns, namespace))
```
`StringEqual()` and `&&` would be less confusing in my opinion.